channels/cz.m3u

```diff
@@ -5,21 +5,23 @@ https://insysgo.live.rd.insyscd.net/echo24.720.smil/manifest.mpd
 http://rtmp.elektrika.cz/live/myStream.sdp/playlist.m3u8
 #EXTINF:-1 tvg-id="ExtasyTVHD.cz" tvg-name="Extasy TV HD" tvg-country="CZ" tvg-language="Czech" tvg-logo="" group-title="XXX",Extasy TV HD
 http://213.151.233.20:8000/dna-6233-tv-pc/hls/4004v105.m3u8
-#EXTINF:-1 tvg-id="Nova Action-CZ" tvg-name="Nova Action-CZ" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/d/d2/Nova-action-logo-2017.png" group-title="",Nova Action
-http://92.62.234.223/102/mystream.m3u8
+#EXTINF:-1 tvg-id="Nova Action.CZ" tvg-name="Nova Action-CZ" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/d/d2/Nova-action-logo-2017.png" group-title="",Nova Action
+https://nova-live.ssl.cdn.cra.cz/channels/nova_action_avod/playlist/cze/live_hq.m3u8
+#EXTINF:-1 tvg-id="Nova Cinema.CZ" tvg-name="Nova Cinema-CZ" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://static.epg.best/cz/NovaCinema.cz.png" group-title="",Nova Cinema
+https://nova-live.ssl.cdn.cra.cz/channels/nova_cinema_avod/playlist/cze/live_hq.m3u8
+#EXTINF:-1 tvg-id="Nova Gold.CZ" tvg-name="Nova Gold-CZ" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://static.epg.best/cz/NovaGold.cz.png" group-title="",Nova Gold
+https://nova-live.ssl.cdn.cra.cz/channels/nova_gold_avod/playlist/cze/live_hq.m3u8
+#EXTINF:-1 tvg-id="Nova.CZ" tvg-name="Nova" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://static.epg.best/cz/Nova.cz.png" group-title="",Nova
+https://nova-live.ssl.cdn.cra.cz/channels/nova_avod/playlist/cze/live_hq.m3u8
+#EXTINF:-1 tvg-id="Nova2.CZ" tvg-name="Nova 2" tvg-country="CZ" tvg-language="Czech" tvg-logo="https://static.epg.best/cz/Nova2.cz.png" group-title="",Nova 2
```

Comment:
prosím o kanaly jako je NOVA, NOVA GOLD, udelat nejakou složku . mě tohle po zkopírování nefunguje

Reply:
Prosím, můžete někdo udělat soubor pro aplikaci FERMATA.

Please can someone make a file for FERMATA application.